pdns/recursordist/docs/settings.rst

```diff
@@ -252,6 +252,16 @@ Which domains we only accept delegations from (a Verisign special).
 Turn off the packet cache. Useful when running with Lua scripts that can
 not be cached.
 
+.. _setting-disable-real-memory-usage:
+.. versionadded:: 4.1.13
```

Comment:
I'm sorry, I suggested the edit on the wrong line, it should have been below the `-----------------------` line following the  `disable-real-memory-usage` one..